.github/workflows/docker_compliance.yml

```diff
@@ -0,0 +1,29 @@
+name: check docker files compliance
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/docker_compliance.yml
+      - '**Dockerfile**'
+  push:
+    branches:
+      - main
+      - 'release/*'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: hadolint/hadolint@sha256:27173fe25e062448490a32de410c08491c626a0bef360aa2ce5d5bdd9384b50d #2.12.0-debian
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+
+      - name: Lint All Dockerfiles (except third_party)
+        run: hadolint -V `find -name "*Dockerfile*" -not -path "./third_party/*" |xargs `
+      - name: Slack Notification
+        if: ${{ failure() && github.ref == 'refs/heads/main' }}
+        continue-on-error: true
+        uses: rtCamp/action-slack-notify@4e5fb42d249be6a45a298f3c9543b111b02f7907
+        env:
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_MESSAGE: "lint finished with status: ${{ job.status }}. (${{ env.ACTION_RUN_URL }})"
```

Comment:
"Docker files lint finished [...]"

Reply:
see previous, worflow name is already included in slack notif